crates/air-lib/air-parser/src/parser/lexer/call_variable_parser.rs

```diff
@@ -186,15 +190,34 @@ impl<'input> CallVariableParser<'input> {
         Ok(())
     }
 
-    fn try_parse_as_stream_start(&mut self) -> LexerResult<bool> {
-        let stream_tag = MetTag::from_tag(self.current_char());
-        if self.current_offset() == 0 && stream_tag.is_tag() {
+    fn try_parse_as_tagged_token(&mut self) -> LexerResult<bool> {
```

Comment:
It worth commenting what is "tagged token".

Reply:
Can you clarify, please?

Reply:
Sure. I am going to add comments on this matter. Not sure yet what is a suitable place for the comment.

Reply:
Added the comments near MetTag implementation.